tests/httpInterface.utils.js

```diff
@@ -43,7 +43,8 @@ async function setUpTest(
   const database = client.db(databaseName)
   const collection = database.collection(mongoDBCollectionName)
 
-  await dropCollectionAndInsertFixtures(collection, testFixtures)
+  const resetCollection = (records = testFixtures) => clearCollectionAndInsertFixtures(collection, records)
```

Comment:
Shouldn't include the `await` and `async` keywords?

Smth like:
```suggestion
  const resetCollection = async (records = testFixtures) => await clearCollectionAndInsertFixtures(collection, records)
```

Reply:
Actually, we recently re-reviewed tap documentation and discovered that when a tap instance returns a Promise it is not necessary to call the `end()` function.

### Reference

https://node-tap.org/docs/structure/

![Screenshot from 2023-03-20 09-44-32](https://user-images.githubusercontent.com/3228581/226288964-19f458ed-a9d7-4492-a905-4a0323daa84a.png)

Reply:
Actually it is not necessary to `await` in a function that does only return the a result, nor such function has to be defined as `async`.  
It is sufficient to `await` on outcome of the external function. Maybe this way does not provide a clear function interface (since it is not explicit the fact that a Promise is returned).